website/routes.py

```diff
@@ -802,6 +803,12 @@ def make_url_map(app):
             OsfWebRenderer('profile/personal_tokens_detail.mako', trust=False)
         ),
 
+        Rule(
+            '/<uid>/quickfiles/',
```

Comment:
Use this a list defined in ember_osf_web_views, e.g. `ember_osf_web_views.routes`